package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sass-embedded",
-  "version": "1.0.0-beta.3",
+  "version": "1.0.0-beta.4-dev",
```

Comment:
This isn't technically a valid semantic version... is npm okay with that? Should it be `1.0.0-dev` instead for now?

Reply:
npm doesn't see this until we push a git tag. But good point about the invalid semantics, I'll changed this to `1.0.0.beta.4` without `-dev`